.github/workflows/release-package.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 20
```

Comment:
Did you test this locally? Node has a tendency of 'breaking' between major versions.

Reply:
Yes i actually did test it now (with act), while i previously didn't tested it because I needed to understand the on release mechanism more